app/views/spree/products/_cart_form.html.erb

```diff
@@ -1,83 +1,164 @@
-<template class="availability-template availability-template-not-available-in-currency">
-  <%= render 'cart_form_availability_templates', type: 'not-available-in-currency' %>
+<!-- <template class="availability-template availability-template-not-available-in-currency">
+  <%#= render 'cart_form_availability_templates', type: 'not-available-in-currency' %>
 </template>
 
 <template class="availability-template availability-template-in-stock">
-  <%= render 'cart_form_availability_templates', type: 'in-stock' %>
+  <%#= render 'cart_form_availability_templates', type: 'in-stock' %>
 </template>
 
 <template class="availability-template availability-template-backorderable">
-  <%= render 'cart_form_availability_templates', type: 'backorderable' %>
+  <%#= render 'cart_form_availability_templates', type: 'backorderable' %>
 </template>
 
 <template class="availability-template availability-template-out-of-stock">
-  <%= render 'cart_form_availability_templates', type: 'out-of-stock' %>
+  <%#= render 'cart_form_availability_templates', type: 'out-of-stock' %>
 </template>
 
-<% is_product_available_in_currency = product_available_in_currency? %>
-<% default_variant = default_variant(@variants, @product) %>
+<%# is_product_available_in_currency = product_available_in_currency? %>
+<%# default_variant = default_variant(@variants, @product) %>
 
-<%= form_for :order, html: {
+<%#= form_for :order, html: {
     id: 'add-to-cart-form',
     class: 'add-to-cart-form',
     'data-product-summary': @product_summary.to_json,
     'data-variants': product_variants_matrix(is_product_available_in_currency),
     'data-variant-change-trigger-identifier': variant_change_identifier
   } do |f| %>
   <div id="inside-product-cart-form" data-hook="inside_product_cart_form">
-    <% if is_product_available_in_currency %>
+    <%# if is_product_available_in_currency %>
       <div id="product-price" class="mb-2 text-center text-md-left add-to-cart-form-price" data-hook="product_price">
```

Comment:
verify this class "add-to-cart-form-price"

Reply:
done its also not disturbing price